client/stylesheets/_toCat.scss

```diff
@@ -4330,4 +4330,153 @@ On click the images will be displayed at normal size to complete the effect
 
 
 
+/* Notif */
+
+.notifMainWrap {
+    width: 70%;
+    text-align: left;
+}
+
+.notifLi {
+    width: 100%;
+}
+
+.notifLi ul > li {
+    display: inline-block;
+}
+
+.notifLi {
+    margin: 2px 0;
+    padding: 10px 0;
+    border-bottom: 1px solid #d3d3d7;
+}
+
+.notifImage img, .notifInfo {
+    vertical-align: middle !important;
+}
+
+.notifImage img, {
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+}
+
+.notifInfo ul > li {
+    display: block;
+}
+
+.notifMessage p, .notifDate p {
+    margin: 0 5px;
+}
+
+.notifMessage p span {
+    font-weight: 700 !important;
+}
+
+.notifMessage p a {
+    font-weight: 700 !important;
+    color: #404040 !important;
+}
+
+.notifMessage p a:hover {
+    color: #404040 !important;
+}
 
+.notifMessage p {
+    font-size: 0.9rem;
+    margin-bottom: -5px;
+}
+
+.notifDate p {
+    font-size: 0.7rem;
+    color: #7b7b7b;
+    margin-top: -5px;
+}
+
+
+/* Settings */
+
+.profileSettingsUl > li {
+    display: block;
+}
+
+.profileSettingsLi ul > li {
+    display: inline-block;
+} 
+
+.profileSettingsLi {
+    border-bottom: 1px solid #d3d3d7;
+    padding: 30px 0;
+    margin: 10px 0;
+    width: 70%;
+}
+
+.profileSettingsLi ul li input, .profileSettingsLi ul li h4, .profileSettingsLi ul li img {
+    vertical-align: middle;
+}
+
+.profileSettingsLi ul li input {
+    padding: 10px 12px;
+    width: 600px;
+    border-radius: 4px;
+    border: 1px solid #d3d3d3;
+}
+
+.profileSettingsLi ul li h4 {
+    margin-right: 40px;
+}
+
+.profileSettingsEmail li input {
+    margin-left: 38px;
+}
+
+.profileSettingsLocation li input {
+    margin-left: 14px;
+}
+
+.profileSettingsUsername li input {
+    margin-left: -5px;
+}
+
+.profileSettingsImage {
+    margin-left: 35px;
+}
+
+.profileSettingsPhoto li h4 {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `profileSettingsPhoto` should be written in lowercase with hyphens